.gitlab-ci.yml

```diff
@@ -0,0 +1,90 @@
+# build needs to be able to find the following libraries:
+# - OpenEXR(libHalf.dylib libHalf.a)
+# - HDF5(hdf5.a) 
+# - Alembic (libAlembic.a)
+
+# sourcedir needs submodule initialization (to compile googletest)
+# - git submodule update --init --recursive
+
+image: node:6.10.0
+
+stages:
+  - build
+  - test
+  - push_to_packman_staging
+
+build:osx:
+  stage: build
+  tags:
+    - buildfarm
+    - darwin
+  script:
+    - if [ -d build ] ; then rm -rf build ; fi
+    - deploydir=`pwd`/Plugin/external/ThirdParty/Deploy/MacOS
+    - mkdir build
+    - cd build
+    - cmake ../Plugin -DCMAKE_BUILD_TYPE=Release \
+        -DCMAKE_INSTALL_PREFIX=`pwd` -DCMAKE_PREFIX_PATH=`pwd` \
```

Comment:
INSTALL_PREFIX is where it's going to be installed. Probably you don't need it at all, since we don't need to 'make install'

PREFIX_PATH is where it should be finding the thirdparty libraries